tests/common2/INSTALL.md

```diff
@@ -0,0 +1,55 @@
+# Setup Instructions
+
+Before committing code, ensure the following tools are installed:
+
+## Installation
+
+### 1. Python
+
+Version: Python 3.9 or higher
+Install from: https://www.python.org/downloads/
+
+### 2. pip (Python package manager)
+
+Usually comes with Python
+Verify with: `pip --version`
+
+### 3. pre-commit
+
+Used to run automated checks before commits
+Install via pip: `pip install pre-commit`
+
+### 4. pylint
+
+Python linter used for code quality checks
+Install via pip: `pip install pylint`
+
+### 5. pyenchant
+
+Install pyenchant `pip install pyenchant` required for pylint spelling linters.
```

Comment:
Combine the pip installations in a single step? Or create a `requirements.txt` file? Or even better, create a script tool for setting up the environment required by the git hooks for pre commit checks?